src/service/DatasetEditManager.ts

```diff
@@ -183,7 +191,7 @@ export class DatasetEditManager {
     ) {
         const content = await this.datasetService.getContent(
             host,
-            `${dataset.name}(${member.name})`,
+            ((dataset.dataSetOrganization === DSOrg.PS) ? `${dataset.name}(${member.name})` : `${dataset.name}`),
```

Comment:
Edit member shouldn't edit datasets.

Reply:
This is the US target. Or you are meaning that I have to clone editMember() method?
Thanks.